package.json

```diff
@@ -87,8 +89,11 @@
   },
   "license": "MIT",
   "scripts": {
+    "arduino:compile": "avr-pizza compile -a uno -s test/arduinoTest/arduinoEcho/arduinoEcho.ino -o test/arduinoTest/arduinoEcho",
+    "arduino:diff": "echo 'help me noopkat! && false",
```

Comment:
Oh! That makes sense, no worries. This would involve running avr-pizza programatically, then using `fs` to read the existing `test/arduinoTest/arduinoEcho`, and then comparing contents. 

Alternatively, this might be a good npm pre-publish hook, or git pre-commit hook to just use avr-pizza to produce the file automatically for the person.